Tchap/target.yml

```diff
@@ -340,6 +340,11 @@ targets:
     templates: 
       - BaseTarget
 
+    settings:
+       base:
```

Comment:
Please adapt this config like you did in #731

Reply:
Done. Please Review.